utils.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
-
 // damn it JS
 export function clone(coll) {
   return JSON.parse(JSON.stringify(coll));
```

Comment:
Oh I'm very aware of that =) It probably prevented all sort of bugs during iteration for me. I have a todo to remove those once I'm done.